Tella/Application/TellaApp.swift

```diff
@@ -14,6 +14,7 @@ struct TellaApp: App {
     private var appViewState = AppViewState()
     @Environment(\.scenePhase) var scenePhase
     @UIApplicationDelegateAdaptor(AppDelegate.self) var appDelegate
+    let delayTimeInSecond = 1.0
```

Comment:
I think `delayTimeInSecond` should be used in `DispatchQueue.main.asyncAfter(deadline: .now() + 1, execute: {` ?
and it will be `DispatchQueue.main.asyncAfter(deadline: .now() + delayTimeInSecond, execute: {`

Reply:
hmm yeah i don't remember touching this, probably is from some merge conflict. I will change it to the correct value. Thanks for catching this!